### PR TITLE
Add upper chest

### PR DESCRIPTION
**Pull request by Louka3000**
for https://github.com/SlimeVR/SlimeVR-Server/pull/716

### CODE REVIEW

File: schema/datatypes/datatypes.fbs [hunk 1]
```diff
@@ -96,23 +96,24 @@ enum BodyPart: uint8 {
...
+    LEFT_FOOT = 11,
+    RIGHT_FOOT = 12,
+    LEFT_LOWER_ARM = 13,
+    RIGHT_LOWER_ARM = 14,
+    LEFT_UPPER_ARM = 15,
+    RIGHT_UPPER_ARM = 16,
+    LEFT_HAND = 17,
+    RIGHT_HAND = 18,
+    LEFT_SHOULDER = 19,
+    RIGHT_SHOULDER = 20,
```
Comment (TheDevMinerTV) [inline schema/datatypes/datatypes.fbs:116]: you're not supposed to change the IDs of the enum members.

File: schema/rpc.fbs [hunk 2]
```diff
@@ -223,24 +223,25 @@ enum SkeletonBone: uint8 {
...
+    FOOT_LENGTH = 12,
+    FOOT_SHIFT = 13,
+    SKELETON_OFFSET = 14,
+    SHOULDERS_DISTANCE = 15,
+    SHOULDERS_WIDTH = 16,
+    UPPER_ARM = 17,
+    LOWER_ARM = 18,
+    HAND_Y = 19,
+    HAND_Z = 20,
+    ELBOW_OFFSET = 21,
```
Comment (TheDevMinerTV) [inline schema/rpc.fbs:244]: you're not supposed to change the IDs of the enum members.

**Review by TheDevMinerTV — CHANGES_REQUESTED** [review]

File: schema/datatypes/datatypes.fbs (hunk 1 above)
Comment (Louka3000) [reply schema/datatypes/datatypes.fbs:116]: I would prefer to and it’s not 1.0

File: schema/rpc.fbs (hunk 2 above)
Comment (Louka3000) [reply schema/rpc.fbs:244]: I would prefer to and it’s not 1.0

Comment (TheButlah) [reply schema/rpc.fbs:244]: No. Please don't change them.

**Review by TheDevMinerTV — APPROVED** [review]
LGTM

**Review by TheButlah — APPROVED** [review]

**Pull request merged**